docs/book/src/topics/FAQ.md

```diff
@@ -0,0 +1,33 @@
+# Frequently Asked Questions
+
+## Does CAPZ support Feature X?
+The best way to check if CAPZ supports a feature is by reviewing [the public milestones](https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones). The public milestones will also provide insight into what's coming in the 
```

Comment:
I'd recommend adding something like "All features should be available for managed (AKS) clusters when using the AzureASOManaged API specification since it covers 100% of the API surface coverage area via ASO." If there is no issue for the new feature request on the repository, then likely it does not exist.